README.md

```diff
@@ -54,19 +54,15 @@ Before you begin, you should have already downloaded the Android Studio SDK and
 
 ## Building the Code
 
-1. Clone the repository using HTTP: git clone https://github.com/openMF/mobile-wallet.git
-
-2. Open Android Studio.
-
-3. Click on 'Open an existing Android Studio project'
-
-4. Browse to the directory where you have cloned the mobile-wallet repo and click OK.
-
+1. Fork the repository.
+2. Go to your fork and clone only the dev branch using `git clone -b dev <remote_repo>`.
```

Comment:
Mention that remote_repo url refers to your own fork and not the 'remote repository'

Reply:
Okay I shall put that in brackets